### PR TITLE
[ACM-4111]: Updates to creating a hosted cluster on bare metal

### PR DESCRIPTION
**Pull request by lahinson**
Jira: https://issues.redhat.com/browse/ACM-4111

@rokej @avishayt PTAL at my comment in the doc about creating a hosted cluster on bare metal. I think we can be clearer about what the user is supposed to do. I'd appreciate your feedback. Thanks!

### CODE REVIEW

File: clusters/hosted_control_planes/managing_hosted_bm.adoc [hunk 1]
```diff
@@ -24,6 +24,8 @@ You must configure hosted control planes for bare metal before you can create ho
 
 Verify that you have a default storage class configured for your cluster. Otherwise, you might end up with pending PVCs.
 
+// What exactly should users do in this step? Do they enter the following commands? Or is the following code the output of another step?
```
Comment (avishayt) [inline clusters/hosted_control_planes/managing_hosted_bm.adoc]: Good call - these are commands to run in a shell.  Same goes for the "oc -n ..." command on line 51.

Comment (lahinson) [reply clusters/hosted_control_planes/managing_hosted_bm.adoc]: @avishayt Thanks for the feedback! I made a couple of updates to the file. When you can, PTAL and let me know what you think.

**Comment by avishayt** [comment]
Looks great, thanks!

File: clusters/hosted_control_planes/managing_hosted_bm.adoc [hunk 2]
```diff
@@ -24,6 +24,8 @@ You must configure hosted control planes for bare metal before you can create ho
 
 Verify that you have a default storage class configured for your cluster. Otherwise, you might end up with pending PVCs.
 
+. In a shell, enter the following commands:
```
Comment (oafischer) [inline clusters/hosted_control_planes/managing_hosted_bm.adoc]: You can leave out the shell here, I don't think we use that.

Comment (lahinson) [reply clusters/hosted_control_planes/managing_hosted_bm.adoc]: Done. Thanks @oafischer!

**Review by oafischer — APPROVED** [review]

**Pull request merged**